## E1128 (assignment-from-none)

### :x: Problematic code:

```python
# to be addded
```

### :heavy_check_mark: Correct code:

```python
# to be added
```

### Rationale:

 *Assigning result of a function call, where the function returns None*
  Used when an assignment is done on a function call but the inferred function
  returns nothing but None. This message belongs to the typecheck checker.



### Related resources:

- [Testcases](#)
- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22assignment-from-none%22+OR+%22E1128%22)
